core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -5171,6 +5200,40 @@ private void checkNullOperand(SqlTester tester, String op) {
     }
   }
 
+  @Test public void testToBase64() {
+    tester.setFor(MySqlOperatorTable.TO_BASE64);
+    SqlTester tester1 = mysqlTester();
+    tester1.checkString("to_base64('This is a test String.')",
+            "VGhpcyBpcyBhIHRlc3QgU3RyaW5nLg==",
+            "VARCHAR NOT NULL");
+
+    tester1.checkNull("to_base64(cast(null as varchar(1)))");
+
+    SqlTester tester2 = mysqlTester(SqlConformanceEnum.MYSQL_5);
+    tester2.checkString("to_base64('This is a test String.')",
+            "VGhpcyBpcyBhIHRlc3QgU3RyaW5nLg==",
+            "VARCHAR NOT NULL");
+
+    tester2.checkNull("to_base64(cast(null as varchar(1)))");
+  }
+
+  @Test public void testFromBase64() {
+    tester.setFor(MySqlOperatorTable.FROM_BASE64);
```

Comment:
Is there a test for `from_base64` with invalid base64 string (not empty but incorrect) ?

Reply:
Also there are some [MySQL specifics](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_to-base64) for base64:

* A newline is added after each 76 characters of encoded output to divide long output into multiple lines.
* Decoding recognizes and ignores newline, carriage return, tab, and space.

Should it exactly replicate same functionality ?